canonical-web-and-design/jaas-dashboard
# Panel improvements

## Done

- Remove duplicate `<main>` element from Login component
- Stop panel form overflowing
- Add back the panel box shadow
- Close panel when escape key is clicked

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify as above
